### PR TITLE
security: [snyk] modified .snyk policy file to exclude vendor/github.com/Azure/go-auto…

### PR DESCRIPTION
**Pull request by pacificcode**
modified .snyk policy file to exclude vendor/github.com/Azure/go-autorest/Gopkg.lock

### CODE REVIEW

File: .snyk
Comment (sheldonhull) [inline .snyk:8]: Before we merge this let's figure out why it's not working, since you should only have to specify a folder, not a subfolder. This list should be minimal so lets look at docs and then if that doesn't answer, reach out to security team on why it's not working.

**Review by sheldonhull — CHANGES_REQUESTED** [review]
pending, let's check before merging another rule on why it's not honoring subfolders.

File: .snyk
Comment (pacificcode) [reply .snyk:8]: @sheldonhull security team is aware of this issue. Alexa Kessler did her homework on this issue and verified with with Snyk representatives. She would be the source of authority for any further/specific details. 
This PR's primary function is to test if including this single file in the .snyk policy file "exclude" directive will eliminate the snyk web UI from reporting these "High" level vulnerabilities. FYI, the snyk cli does not report the vulnerabilities being addressed by this PR.

Comment (sheldonhull) [reply .snyk:8]: @pacificcode I'm discussing with Alexa for some context before merging since there's some research I did on this prior that might have relevance. Once I confirm with her I'll either approve or update here with more context, so leave this open temporarily  and I'll resolve it once I get some details to add. Didn't want you to think I'd forgotten :-)

**Review by sheldonhull — CHANGES_REQUESTED** [review]

File: .snyk
Comment (pacificcode) [reply .snyk:8]: @sheldonhull any update on this? it would be great to get it merged.

Comment (sheldonhull) [reply .snyk:8]: same update from 2 weeks ago. I did bump the internal conversation and will update once I have something more, or you can bump that thread as well. 🙇🏻

**Pull request closed without merging**
